tests/ethpm/test_get_deployments.py

```diff
@@ -38,7 +49,8 @@ def test_get_deployments_with_multiple_matches_raises_exception(w3, lockfile_wit
         package.get_deployments(w3)
 
 
-def test_get_deployments_with_a_match_returns_deployments(w3, lockfile_with_matching_deployment):
-    package = Package(lockfile_with_matching_deployment)
-    deployment = package.get_deployments(w3)
+def test_get_deployments_with_a_match_returns_deployments(w3, matching_package):
+    deployment = matching_package.get_deployments(w3)
+    contract_factory = deployment.contract_factories["SafeMathLib"]
     assert isinstance(deployment, Deployments)
+    assert getattr(contract_factory, '__module__') == web3.contract.__name__
```

Comment:
I was trying to come up with a better way to test the contract_factory is a contract_factory, but I couldn't get isinstance() to cooperate. Is this fine / is there a better way?

Reply:
Hrm, maybe it'd be better to `assert "SafeMathLib" in deployments` to simply test for presence.